### PR TITLE
(2) Updated README – information on loading via Zplugin (Zsh)

### PR DESCRIPTION
**Pull request by psprint**
Hello,
I've created a new pull-request as requested in #81 – I couldn't update the old pull request.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -37,6 +37,14 @@ echo 'eval $(dircolors -b $HOME/.dircolors)' >> $HOME/.bashrc
 Archlinux users can install the [`lscolors-git`][3] package from the AUR for easy
 integration with bash, csh, or zsh.
 
+There's a Zsh plugin manager `Zplugin` that nicely works with this repository – `dircolors`
```
Comment (rpdelaney) [inline README.markdown]: Is the idea that the user would install `lscolors-git` from the AUR and then use your plugin? Can't your plugin be used without the AUR package though? I'm suspicious this should either have its own section header -or- get merged with the "ZSH SYNTAX HIGHLIGHTING" section below. What do you think?

**Comment by psprint** [comment]
It fully installs LS_COLORS, Aur package is'nt needed. So it clones the repository to ~/.zplugin/plugins, runs the atclone"" and atpull"" hooks, sources the file generated by the hooks (which run `dircolors`). It is a full installation method.

**Comment by rpdelaney** [comment]
Okay, thanks for explaining. Please add a section header to demarcate this information from the instructions for the AUR package above. Something like `### zsh integration with Zplugin` would be logical.

**Comment by rpdelaney** [comment]
Closing this since it needs updates but there is a merge conflict with master. Please feel free to resubmit when you get the time.

**Pull request closed without merging**